karma/test-directives.coffee

```diff
@@ -60,6 +60,29 @@ describe('directives:', () ->
     )
   )
 
+  #=======================================================================
+  # Tests for tooltip
+  #=======================================================================
+  describe('cpTooltip', () ->
+
+    it('adds an element with a tooltip to the dom', () ->
+      $templateCache.put('/sitestatic/templates/tooltip.html')
+      $scope = $rootScope.$new()
+
+      template = $compile('<cp-tooltip display-text="hello" tooltip-text="world" position="top" ></cp-tooltip>')
```

Comment:
Should we add some content to the element here and test that we display this content?